cpu/esp32/periph/adc.c

```diff
@@ -397,6 +397,7 @@ static void _adc_module_init(void)
 
 #if defined(DAC_GPIOS)
 
+static const unsigned dac_chn_num = DAC_NUMOF;
```

Comment:
`dac_chn_num` is being define two times, also, couldn't `DAC_NUMOF` be used instead?